examples/visual-tests/amp-story/amp-story-shopping-lang-de.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<html amp lang="de">
```

Comment:
It might be good to add a comment before line 2 that explains the reason for the visual test.